Broken io.prestosql.tests.hive.TestHiveTransactionalTable.testReadFullAcid

```
2020-06-19T06:36:14.6669489Z tests               | 2020-06-19 12:21:14 INFO: FAILURE     /    io.prestosql.tests.hive.TestHiveTransactionalTable.testReadFullAcid (Groups: storage_formats, hive_transactional) took 50.4 seconds
2020-06-19T06:36:14.6738598Z tests               | 2020-06-19 12:21:14 SEVERE: Failure cause:
2020-06-19T06:36:14.6738853Z tests               | java.lang.AssertionError: tableCompactionsIds are empty expected [false] but found [true]
2020-06-19T06:36:14.6776353Z tests               | 	at org.testng.Assert.fail(Assert.java:94)
2020-06-19T06:36:14.6776903Z tests               | 	at org.testng.Assert.failNotEquals(Assert.java:513)
2020-06-19T06:36:14.6777233Z tests               | 	at org.testng.Assert.assertFalse(Assert.java:63)
2020-06-19T06:36:14.6777581Z tests               | 	at io.prestosql.tests.hive.TestHiveTransactionalTable.compactTableAndWait(TestHiveTransactionalTable.java:203)
2020-06-19T06:36:14.6777931Z tests               | 	at io.prestosql.tests.hive.TestHiveTransactionalTable.testReadFullAcid(TestHiveTransactionalTable.java:95)
2020-06-19T06:36:14.6778304Z tests               | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
2020-06-19T06:36:14.6778641Z tests               | 	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
2020-06-19T06:36:14.6778981Z tests               | 	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
2020-06-19T06:36:14.6779311Z tests               | 	at java.base/java.lang.reflect.Method.invoke(Method.java:566)
2020-06-19T06:36:14.6779643Z tests               | 	at org.testng.internal.MethodInvocationHelper.invokeMethod(MethodInvocationHelper.java:104)
2020-06-19T06:36:14.6779973Z tests               | 	at org.testng.internal.InvokeMethodRunnable.runOne(InvokeMethodRunnable.java:54)
2020-06-19T06:36:14.6780297Z tests               | 	at org.testng.internal.InvokeMethodRunnable.run(InvokeMethodRunnable.java:44)
2020-06-19T06:36:14.6780879Z tests               | 	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:515)
2020-06-19T06:36:14.6781211Z tests               | 	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
2020-06-19T06:36:14.6781543Z tests               | 	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1128)
2020-06-19T06:36:14.6781877Z tests               | 	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:628)
2020-06-19T06:36:14.6782217Z tests               | 	at java.base/java.lang.Thread.run(Thread.java:834)
```

--- a/presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java
+++ b/presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveTransactionalTable.java
@@ -181,10 +181,11 @@ public class TestHiveTransactionalTable
 
     private static String hiveTableProperties(TransactionalTableType transactionalTableType, BucketingType bucketingType)
     {
-        return Stream.concat(
-                transactionalTableType.getHiveTableProperties().stream(),
-                bucketingType.getHiveTableProperties().stream())
-                .collect(joining(",", "TBLPROPERTIES (", ")"));
+        ImmutableList.Builder<String> tableProperties = ImmutableList.builder();
+        tableProperties.addAll(transactionalTableType.getHiveTableProperties());
+        tableProperties.addAll(bucketingType.getHiveTableProperties());
+        tableProperties.add("'NO_AUTO_COMPACTION'='true'");
+        return tableProperties.build().stream().collect(joining(",", "TBLPROPERTIES (", ")"));
     }
 
     private static void compactTableAndWait(CompactionMode compactMode, String tableName, String partitionString, Duration timeout)

